packages/admin/src/router/ConfirmationDialog.tsx

```diff
@@ -16,9 +16,10 @@ interface Props {
     isOpen: boolean;
     message: React.ReactNode; // typically a string or a FormattedMessage (intl) is passed
     handleClose: (action: PromptAction) => void;
+    showSaveButton: boolean;
```

Comment:
```suggestion
    showSaveButton?: boolean;
```

You're providing a default value for `showSaveButton`, so why not make the prop optional?